limitDockerNodeUpdatesToLTS.json5

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "regexManagers": [
+    {
+      "fileMatch": [
+        "^*Dockerfile*$"
```

Comment:
- `^*` seems invalid
- `e*`: 0 and unlimited `e` character is really expected? `Dockerfil` and `Dockerfileee` would match

Reply:
Nice catch, thank you. I will adjust the `fileMatch` .

One question, if I have something like `devel.Dockerfile` for example,
`"(^|/)Dockerfile-[^/]*$"`  this regex does not match right? 🤔

Reply:
> if I have something like `devel.Dockerfile` for example,
`"(^|/)Dockerfile-[^/]*$"` this regex does not match right?

That's right, but the case should match with `'(^|/|\\.)Dockerfile$'`.

Reply:
Ah, I think I understand now. This repo's dockerfile setting extends the default config so that it matches `Dockerfile-foo` as well. Ok.

Reply:
Yes, `fileMatch` is *mergeable* config, so our `dockerfile.fileMatch` extends the default config.

https://docs.renovatebot.com/configuration-options/#filematch

Reply:
I added the regular expressions given above for `fileMatch` https://github.com/hatena/renovate-config/pull/62/commits/3cb5bc507f6b5c2b7256524d85f1722ce66a08e6